data_management_scripts/cleanup_dataset_matrixtables.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3  # noqa: EXE001
+
+import logging
+import subprocess
+import sys
+from datetime import datetime
+
+import click
+from cpg_utils.config import get_config
+from google.cloud import storage
+
+GENOME_PREFIX = 'mt/'
+EXOME_PREFIX = 'exome/mt/'
+
+
+def get_dataset_mt_timestamps(
```

Comment:
I very much thought this should be coming from Metamist. I was today years old when I realised that we've never registered these as Analysis entries 😞